### PR TITLE
Add Mailjet smtp integration to documentation

### PR DESCRIPTION
**Pull request by npracht**
Documentation related to https://github.com/mautic/mautic/pull/2346

### CODE REVIEW

File: en/emails/bounce_management.md [hunk 1]
```diff
@@ -32,28 +32,44 @@ This is not required, but if you'll want to be able to select the contacts with
...
 
 5) Click Custom Metadata and create two new metadata fields: `hashId` and `contactId`
 
 ![Add metadata](/emails/media/mandrill_webhook_5.png "Add metadata")
 
 ![Add metadata](/emails/media/mandrill_webhook_4.png "Add metadata")
 
+## Mailjet Webhook configuration
+
+Mautic supports Mailjet's webhooks for bounces, spam and blocked. Before any configuration, you'll need to create an account on [Mailjet](http://www.mailjet.com/?aff=mauticwm).
```
Comment (alanhartless) [inline en/emails/bounce_management.md]: Please remove the affiliate parameter. Thanks!

**Comment by npracht** [comment]
I think i'm good @alanhartless :)

**Comment by alanhartless** [comment]
Thanks! Can you look at the associated PR in mautic/mautic because I moved it to 2.3.0 because it had a code question and wasn't for sure if you guys would be able to get to it before 2.2.0 release. If it can be addressed today - we'll move it back.

**Comment by npracht** [comment]
Hi @alanhartless i think we're now up to date :)

**Pull request merged**